test/app/model/King.spec.ts

```diff
@@ -0,0 +1,60 @@
+import 'jest-extended';
+import { boardWithPieces, emptyBoard } from './BoardFixture';
+import { Side, Square } from '../../../src/app/model/Types';
+import { King } from '../../../src/app/model/King';
+import { Pawn } from '../../../src/app/model/Pawn';
+import { Knight } from '../../../src/app/model/Knight';
+
+describe("King's possible moves", () => {
+  it('When king is on E1, not surrounded by any piece and all possible moves are legal (none of squares is checked)', () => {
+    const whiteKing = new King(Side.WHITE);
+    const whiteKingPosition: Square = { column: 'E', row: 1 };
+    const expectedKingPossibleMoves = [
```

Comment:
Jedyne co to te expected proponowalbym przeniesc zaraz nad 
`expect(kingPossibleMoves).toIncludeSameMembers(expectedKingPossibleMoves);` bo to taka czesc THEN jakbys sobie podzielila, a na górze jest GIVEN.